content/blog/configuring-azure-ad-with-long-term-token-for-scim-provisiong.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Configuring Azure Active Directory with long-lived tokens for user provisioning
+date: 2024-03-05T22:06:07.623Z
+priority: 9
+author: Meena Krishnamoorthy
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
+  - hpe-greenlake-for-private-cloud-enterprise
+---
+Azure Active Directory (Azure AD) is Microsoft's cloud-based identity and access management service, designed to simplify user authentication and authorization across various applications and platforms. It offers a centralized solution for managing user identities, enforcing security policies, and facilitating seamless access to cloud-based resources. Azure AD automatic user provisioning simplifies the creation, maintenance, and removal of user identities in SaaS applications based on business rules.
+
+The Azure AD provisioning service provisions users to HPE GreenLake portal by connecting to the user management API endpoints provided by HPE GreenLake Identity and Access Management (IAM). These user management API endpoints allow Azure AD to programmatically create, update, and remove users and groups. The Azure AD provisioning service uses HPE GreenLake tenant API token to provision users and groups to HPE GreenLake IAM.  The HPE tenant API tokens are only valid for fifteen minutes. Because Azure AD cannot automatically renew the token, long-term tokens are required.
+
+I'll explain the process for configuring Azure AD to use a long-term token for user and group provisioning.
+
+## S﻿teps to configure long-term token in Azure AD
+
+1. A﻿pply System for Cross-domain Identity Management(SCIM) proxy token contributor role to IAM user/group
+2. G﻿et a personal access token
+3. C﻿reate a SCIM proxy token
+4. U﻿pdate the SCIM proxy token and the tenant URL in Azure AD Enterprise Application
+
+## S﻿tep 1: Apply System for Cross-domain Identity Management(SCIM) proxy token contributor role to IAM user/group
+
+A﻿ssign "SCIM Proxy Token Contributor" role to the user or user group that will create the long-term token
+
+* L﻿og in to the HPE GreenLake Cental.
```

Comment:
Change "in to" to "into".
Check the spelling of "Central"

Reply:
@meenakris 
Could you please check the spelling of HPE GreenLake Central in this sentence: Log in to the HPE GreenLake Central.

**Furthermore,** the **_HPE GreenLake Central_** is now known as **_HPE GreenLake Flex Solutions_**.
You may want to adjust the occurrences of "HPE GreenLake Central" in your blog post to "HPE GreenLake Flex Solutions".

There is a bullet item that mention HPE GreenLake Central in:

**Step 1: apply System for Cross-domain Identity Management (SCIM) proxy token contributor role to IAM user/group**

and

**Step 2: Get a personal access token**

You may want to adjust the text of the first bullet item to something like that:

- Log in to the HPE GreenLake Central, also known as HPE GreenLake Flex Solutions.

If you have an HPE account associated to a HPE GreenLake Workspace, you can log in to the common.cloud.hpe.com, and in the "Services", in the service catalog, you will see information about the "**HPE GreenLake Flex solutions**":

_Formerly known as HPE GreenLake Central, view usage and cost of HPE GreenLake Flex Solutions and manage the Private Cloud Enterprise and High Performance Computing cloud services. For all HPE GreenLake Flex Solutions, this service provides reporting on billing, active capacity planning, and powerful consumption analytics that enable transparency to see costs by service type, department, or other parameters, including public cloud costs. Customers of Private Cloud Enterprise use it to provision and manage virtual, containerized, and bare metal resources, and customers of High Performance Computing use it to operate HPC workloads._